Gemfile

```diff
@@ -64,6 +64,7 @@ gem 'lograge', '~> 0.11.2'
 gem 'okcomputer'
 gem 'pg'
 gem 'propshaft'
+gem 'pry-rails'
 gem 'redis', '~> 4.0'
```

Comment:
This is preferable to irb when debugging -- I was having some odd problems when testing queries.

Reply:
Should this go in the development group?  Should we consider this change in a separate PR?

Reply:
Yeah, I'll pull into a separate PR.  My first inclination was also to put in the development group, though other apps seem to not do this, e.g.

https://github.com/sul-dlss/workflow-server-rails/blob/main/Gemfile#L17
https://github.com/sul-dlss/pre-assembly/blob/main/Gemfile#L13

and others: https://github.com/search?q=org%3Asul-dlss+pry-rails&type=code